code/modules/client/client procs.dm

```diff
@@ -20,6 +20,34 @@
 		- If so, is there any protection against somebody spam-clicking a link?
 	If you have any  questions about this stuff feel free to ask. ~Carn
 	*/
+
+/client/MouseDown(object,location,control,params)
+	if(mob && prefs && prefs.autoclick_delay && object && istype(mob, /mob/living/carbon))
+		var/mob/living/carbon/C = mob
+		if(!C.get_active_hand())
+			return ..()
+		C.is_mouse_down = object
+		spawn()
+			while(C.is_mouse_down == object)
+				C.ClickOn(C.is_mouse_down, params)
+				sleep(1)

```

Comment:
This should be sleep(world.tick_lag) so it'll work correctly if we move to 20+ fps.